packages/opentelemetry-host-metrics/src/BaseMetrics.ts

```diff
@@ -39,18 +37,18 @@ const DEFAULT_NAME = '@opentelemetry/host-metrics';
  * Base Class for metrics
  */
 export abstract class BaseMetrics {
-  protected _logger = api.diag;
-  protected _meter: api.Meter;
+  protected _logger = diag;
+  protected _meter: Meter;
   private _name: string;
 
   constructor(config: MetricsCollectorConfig) {
     this._name = config.name || DEFAULT_NAME;
     const meterProvider =
-      config.meterProvider || api.metrics.getMeterProvider();
+      config.meterProvider || metrics.getMeterProvider();
     if (!config.meterProvider) {
       this._logger.warn('No meter provider, using default');
     }
-    this._meter = meterProvider.getMeter(this._name, VERSION);
+    this._meter = meterProvider.getMeter(this._name);
```

Comment:
Although is optional I think the `VERSION` here is useful. If I'm not mistaken this info will be added as metadata in the metrics so we will know which version of the package sent the data. There might be use cases for this

Reply:
Reverted